kernel/packages/shared/meta/selectors.ts

```diff
@@ -33,16 +33,5 @@ export const isMOTDInitialized = (store: RootMetaState): boolean =>
 export const getMessageOfTheDay = (store: RootMetaState): MessageOfTheDayConfig | null =>
   store.meta.config.world ? store.meta.config.world.messageOfTheDay || null : null
 
-export const isVoiceChatEnabled = (store: RootMetaState): boolean => {
-  if (!WORLD_EXPLORER || VOICE_CHAT_DISABLED_FLAG) return false
-  return !!getCommsConfig(store).voiceChatEnabled || VOICE_CHAT_ENABLED_FLAG
-}
-
-export const getVoiceChatAllowlist = (store: RootMetaState): string[] => getCommsConfig(store).voiceChatAllowlist ?? []
-
-export const isVoiceChatEnabledFor = (store: RootMetaState, userId: string): boolean =>
```

Comment:
I'd also leave this method. If for any reason we need to have an enable/disable functionality for voice chat in the future, we can use this as hook.

Reply:
Done!